packages/dso-toolkit/src/components/form/form-groups/input-number/form-group-input-number.scss

```diff
@@ -18,3 +18,11 @@
     }
   }
 }
+
+.dso-input-number:not(.form-group) {
```

Comment:
Deze selector gaat over het component Input Number en staat in de stylesheet die hoort bij het component Form Group: Input Number.

Gisteren hebben we toch besproken dat Form Group: Input Number ook wordt verwijderd?

Reply:
Echt? Geheugen als een zeef.

Reply:
Ik heb Form Group: Input Number nu verwijderd.